Mathlib/Analysis/NormedSpace/ENormedSpace.lean

```diff
@@ -108,14 +112,14 @@ theorem map_sub_le (x y : V) : e (x - y) ≤ e x + e y :=
     _ ≤ e x + e (-y) := e.map_add_le x (-y)
     _ = e x + e y := by rw [e.map_neg]
 
-instance partialOrder : PartialOrder (ENorm 𝕜 V) where
+instance partialOrder : PartialOrder (ENormedSpace 𝕜 V) where
```

Comment:
rename this instance to match the conventions (e.g. `instPartialOrder`), while at it?